ci/infra/testrunner/README.md

```diff
@@ -69,21 +219,71 @@ export VSPHERE_PASSWORD="password"
 export VSPHERE_ALLOW_UNVERIFIED_SSL="true"
 ```
 
-2. Set `VMWARE_ENV_FILE=/path/to/vmware-env.sh`
-
-or
+2. Set the path to the VMware environment file in the testrunner's vars file:
 
 ```
 vmware:
-  env_file: ""
+  env_file: "=/path/to/vmware-env.sh"
```

Comment:
```suggestion
  env_file: "/path/to/vmware-env.sh"
```
I assume the leading `=` is a typo, since it's not in the environment variable below?